airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -15,13 +15,13 @@
 from pytimeparse import parse as pytime_parse
 from sqlalchemy import Column, String, and_, func
 from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy_utcdatetime import UTCDateTime
 
 from airflow.configuration import conf
 from airflow.models import DagModel, DagRun, TaskFail, TaskInstance, XCom
 from airflow.plugins_manager import AirflowPlugin
 from airflow.settings import RBAC, Session
 from airflow.utils.log.logging_mixin import LoggingMixin
-from airflow.utils.sqlalchemy import UtcDateTime
```

Comment:
It's for local dev purpose. So we don't have to include airflow utils but sqlalchemy_utcdatetime.